torchtune/datasets/_instruct.py

```diff
@@ -18,8 +18,10 @@
 )
 from torchtune.datasets._packed import PackedDataset
 from torchtune.modules.tokenizers import ModelTokenizer
+from torchtune.utils.logging import deprecated
 
 
+@deprecated(msg="Please use `torchtune.datasets.SFTDataset` for custom chat data.")
```

Comment:
I think @ebsmothers already raised this, but worth pointing the user to a concrete example of a replacement?
If not, this says "chat" instead of instruct btw